Garbled Circuit.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Garbled Circuit
+parent: Protocols
+grand_parent: Contents
+has_children: true
+nav_order: 1
+---
+
+# Secure Multi-Party Computation from Garbling
+
+
+Author: Xiang Xie @PlatON
+
+This post was initially published in [PlatON](https://www.platon.network/news/4?newsType=2).
+
+As described in the previous article, secure multi-party computation protocols are distributed protocols that allow participated players to jointly compute some pre-agreed function without revealing the privacy of each input information. PlatON provides an end-to-end MPC framework to enable privacy-preserving computation. In this article, we will get into more details on the techniques to show how it works.
+
+Before technique details, one should define the security goal, which is called the security model in cryptography. The semi-honest model and malicious model are widely considered to capture the capabilities of adversaries.
+
+In the semi-honest setting, the players strictly follow the instructions of the protocol, and are still curious to learn information of other inputs from the transactions. This is meaningful in some enterprise-to-enterprise business scenarios, where the enterprises must strictly follow the procedure and the codes are run in some safe place that stuffs can not touch. The malicious setting is more realistic, which means the players could do what they can to get private information of other inputs. A secure MPC protocol means that in both setting, all the players can not learn any additional information of the outputs. Note that "additional information" here does not mean "any information" of the input, anything could directly derived from the output result is allowed to be learned by the players.

```

Comment:
I suggest to have a separate entry about semi-honest and about malicious - and copy the definitions from here. Then we can link to those pages instead of writing the full definition here.

Reply:
Agree, this makes sense.